driver/terraform_test.go

```diff
@@ -107,6 +110,53 @@ func TestRenderTemplate(t *testing.T) {
 	}
 }
 
+func TestTerraform_Version(t *testing.T) {
+	TerraformVersion = &goVersion.Version{}
```

Comment:
What is this line for?

Reply:
I'm initializing the global variable

Reply:
I'll add a comment, I'll also improve this:

```Go
func TestTerraform_Version(t *testing.T) {
	var err error
	TerraformVersion, err = goVersion.NewVersion("1.2")
	require.NoError(t, err)
	var tf Terraform
	s := tf.Version()
	assert.Equal(t, "1.2.0", s)
}
```